tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -422,6 +422,53 @@ DISABLED_DRAW_TEST_F(CGContext, DrawAnImage, UIKitMimicTest) {
     CGContextDrawImage(context, bounds, image.get());
 }
 
+DRAW_TEST_F(CGContext, DrawAnImageWithOpacity, UIKitMimicTest) {
+    // Load an Image and draw it into the canvas context
+    auto drawingConfig = DrawingTestConfig::Get();
+
+    woc::unique_cf<CFStringRef> testFilename{ _CFStringCreateWithStdString(drawingConfig->GetResourcePath("png3.9.png")) };
```

Comment:
hmm actually this is from an earlier naming convention. to indicate the file name.
although it should be png1.9 here.

---
In reply to: [93539042](https://github.com/Microsoft/WinObjC/pull/1557#discussion_r93539042) [](ancestors = 93539042,93538951)